docs/content/preview/reference/configuration/yb-master.md

```diff
@@ -159,6 +159,20 @@ Location of .htpasswd file containing usernames and hashed passwords, for authen
 
 Default: `""`
 
+##### --defer_index_backfill
+
+If enabled, will avoid launching any new index-backfills on the cluster for all new YCQL. 
```

Comment:
```suggestion
If enabled, yb-master avoids launching any new index-backfills on the cluster for all new YCQL tables. 
```

Reply:
not sure if this is what needs to be added, but it feels incomplete.

Reply:
See https://github.com/yugabyte/yugabyte-db/pull/20523/commits/a82782f2eca7bcf379d4c6014d38fa4394dcf0a7